lg_activity/scripts/write_scene_timestamps.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import os
+import time
+import json
+import rospy
+from interactivespaces_msgs.msg import GenericMessage
+
+
+IGNORED_SCENE_SLUGS = ["auto_generated_sv_scene"]
+
+def save_scene_played_time(data):
+    """Callback for /director/scene listener, updates the played time for the scene."""
+    #input_file_path = '/media/videos/scene_times.json'
+    scene_times_path = '/mnt/videos/scene_times.json'
+    scene_infos_path = '/mnt/videos/scene_infos.json'
+    if os.path.exists(scene_times_path):
+        with open(scene_times_path, 'r') as times_file:
+            scene_times = json.load(times_file)
+    else:
+        scene_times = []
+    try:
+        message_dict = json.loads(data.message)
+    except:
+        print("failed to load message")
+        return
+
+    slug = None
+    if message_dict:
+        slug = message_dict.get('slug', None)
+    else:
+        print("failed to load: ", data)
+        return
+
+    if slug:
```

Comment:
seems like you can have `if not slug or slug in IGNORED_SCENE_SLUGS: print & return` to avoid a bit of checking and indentation below